### PR TITLE
Rename Gradle Enterprise to Develocity

### PR DESCRIPTION
**Pull request by welandaz**
This PR renames Gradle Enterprise to Develocity in README (incl. screenshots), UI elements and parts of the code.

Environment variables that are used for autoinjection were not renamed.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -14,8 +14,8 @@ http://www.gradle.org/[Gradle] is managed as another tool inside Jenkins (the sa
 
 It also allows detecting https://gradle.com/build-scans/[Build Scans] in arbitrary console logs, for Maven and Gradle builds and display them in the Jenkins UI.
```
Comment (guylabs) [inline README.adoc:15]:
```suggestion
It also allows detecting https://gradle.com/build-scans/[Build Scan] links in arbitrary console logs, for Maven and Gradle builds and display them in the Jenkins UI.
```

Can you check and rename all occurrences of `Build Scans` to `Build Scan`? If we still need the plural version, do not write it in upper-case.

File: images/enriched-build-scans-configuration.png
Comment (guylabs) [inline images/enriched-build-scans-configuration.png:1]: Use `Build Scan` instead of `Build Scans`.

**Review by guylabs — APPROVED** [review]

File: README.adoc (hunk 1 above)
Comment (welandaz) [reply README.adoc]: Done

File: images/enriched-build-scans-configuration.png
Comment (welandaz) [reply images/enriched-build-scans-configuration.png:1]: Done

**Review by alextu — APPROVED** [review]
LGTM, although:
- one acceptance test is failing
- we should test the upgrade path

**Comment by alextu** [comment]
I've tested the upgrade path:
- given installed plugin 2.9 version
- checked on maven freestyle, gradle freestyle, gradle pipeline jobs that injection works
- manually deployed build from this branch
- restarted
- checked visually that "Develocity" appears in the UI
- checked on existing maven freestyle, gradle freestyle, gradle pipeline jobs that injection still works

**Pull request merged**